src/notte/browser/pool/local_pool.py

```diff
@@ -123,7 +123,11 @@ def get_chromium_args(self, cdp_port: int) -> list[str]:
                     f"--custom-devtools-frontend={self.custom_devtools_frontend}",
                 ]
             )
-        return chromium_args + [port]
+
+        if cdp_port is not None:
+            chromium_args.append(f"--remote-debugging-port={cdp_port}")
```

Comment:
you need to make a copy of chronium args otherwise you might run into issues. but actually I think I would opt for not storing `self.chromium_args` at all